### PR TITLE
Use Case Survey Notice

### PR DESCRIPTION
**Pull request by ogi-m**
Submitting against the release branch, [original PR](https://github.com/mattermost/notices/pull/292#issuecomment-1227512374) was submitted against the master branch.

### CODE REVIEW

File: notices.json [hunk 1]
```diff
@@ -1,5 +1,24 @@
...
+      "audience": "all",
+      "clientType": "all",
+      "instanceType": "all",
+      "repeatable": "false",
+      "displayDate": "> 2022-08-01T00:00:00Z <= 2022-09-30T00:00:00Z"
+    },
+    "localizedMessages": {
+      "en": {
+        "title": "We want your feedback!",
+        "description": "We are looking for feedback on how we are doing in supporting your team's collaboration throughout different phases of the DevOps lifecycle. Please help us with a quick 5 minutes survey and a chance to win a set of AirPod Pros.",
```
Comment (esethna) [inline notices.json:14]: Propose: and a chance... > for a chance...

**Review by esethna — DISMISSED** [review]

File: notices.json [hunk 2]
```diff
@@ -2,9 +2,6 @@
     {
     "id": "use_case_survey",
     "conditions": {
-      "audience": "all",
-      "clientType": "all",
-      "instanceType": "all",
```
Comment (ogi-m) [inline notices.json]: Removed redundant conditions that default to "all" when not explicitly stated.

**Comment by ogi-m** [comment]
I couldn't get the date range to work. I'll file a ticket for that, but in the meantime I propose removing the end date from the date condition and removing the notice altogether from the `notices.json` file on the appropriate day. Thoughts @wiersgallak ?

File: notices.json (hunk 1 above)
Comment (wiersgallak) [reply notices.json]: Updated

File: notices.json (hunk 2 above)
Comment (wiersgallak) [reply notices.json]: Thanks!

**Comment by wiersgallak** [comment]
@ogi-m Sounds good to me. I removed the end date and updated the start date to be >= .  Please let me know if there are any other issues on the formatting.

**Comment by ogi-m** [comment]
Thanks @wiersgallak!. I've opened [a ticket](https://mattermost.atlassian.net/browse/MM-46960) to track the date range issue.

@esethna Could you please merge this notice? Thanks

**Review by esethna — APPROVED** [review]

**Pull request merged**